### PR TITLE
feat: remove instructions and hyperdev

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39720

Linked to learn PR: https://github.com/freeCodeCamp/freeCodeCamp/pull/39970

Cannot be merged, before 39970 hits production

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -3,28 +3,29 @@
  * the verification process may break
  *******************************************/
 
-var express = require('express');
-var app = express();
-try{
-  var mongoose = require('mongoose');
```
Comment (moT01) [inline server.js:9]: I noticed that changing this from var to const broke the test for me when running locally. Were you able to get this to pass? Basically, I forked this repo and checked out this branch, got it running - tried to do the first lesson and it didn't pass. The test goes to `URL/_api/is-mongoose-ok` and it was giving me back a `mongoose is not defined`, when I changed this back to `var` it gave the expect `isMongooseOk: true`. 

I'm curious if you want to give that a try @Sky020 and see if you have the same issue. If you do, I'm wondering if we should maybe not change these files on the boilerplates and leave them how they were.

Comment (ShaunSHamilton) [reply server.js:9]: @moT01 Thank you, for catching that. It was poor diligence on my part, and appear to have missed a few things.

I will push the necessary changes, soon.

Comment (ShaunSHamilton) [reply server.js:9]: @moT01 Thank you, for catching that. It was poor diligence on my part, and appear to have missed a few things.

I will push the necessary changes, soon.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

File: views/index.html [hunk 2]
```diff
@@ -28,20 +31,13 @@
 
   <body>
     <header>
```
Comment (RandellDawson) [inline views/index.html]: Can we just remove lines 33 through 45 for now and replace with:
```
    <h1>MongoDB & Mongoose</h1>
```
Later in a separate PR, we can always add a `header`, `main`, and `footer`, but I can not think of anything to put here and we definitely do not need that footer.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
One small suggestion.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Once the changes from PR #39970 hit production, we can merge this one.

**Comment by RandellDawson** [comment]
Thank you @Sky020 for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Comment by IC-XC** [comment]
When I start this course and I want to start with "Install and Set Up Mongoose", if I use "Use our Replit starter project" I get " Error: Cannot find module 'express'

**Comment by ShaunSHamilton** [comment]
@IC-XC I suggest creating a topic on the [forum](https://forum.freecodecamp.org/) asking for help.

It is likely you are coming across an issue with Replit where it is no longer installing the packages when Run is first clicked. In which case, you need to manually install the packages with `npm i`.

**Pull request merged**